test/e2e/specs/editor/various/rtl.spec.js

```diff
@@ -41,21 +39,38 @@ describe( 'RTL', () => {
 
 		// Expect: ARABIC_ZERO + ARABIC_ONE + ARABIC_TWO (<p>٠١٢</p>).
 		// N.b.: HTML is LTR, so direction will be reversed!
-		expect( await getEditedPostContent() ).toMatchSnapshot();
+
+		//Check the content
```

Comment:
Nit: 
```suggestion
		// Check the content.
```

and ditto to the others below.